docs/source/installation/troubleshooting.rst

```diff
@@ -182,3 +182,108 @@ operating system's documentation to find out where you can get it and how you
 have to install it.
 
 If you are unable to solve your problem, check out the `dvisvgm FAQ <https://dvisvgm.de/FAQ/>`_.
+
+(Windows) ``Python is not recognized as an internal or external command, operable program or batch file.``
+--------------------------------------------------------------------------------------------------------
+
+1. Open System Properties (Right click Computer in the start menu, or use the keyboard shortcut Win+Pause)
+2. Click Advanced system settings in the sidebar.
+3. Click Environment Variables...
+4. Select PATH in the System variables section
+5. Click Edit
+6. Add Python's path to the end of the list (the paths are separated by semicolons). For example:
+
+.. code-block:: bash
+
+  C:\Windows;C:\Windows\System32;C:\Python27
```

Comment:
I feel like it would be more informative to state what the path to python might look like (with the note that the user's python's path might not be exactly the same).